ST_grid.c

```diff
@@ -1340,7 +1349,7 @@ void _Output_AllCellAvgBmass(const char * filename){
 		  grpsos[SuperGlobals.max_rgroups], gsize[SuperGlobals.max_rgroups], gpr[SuperGlobals.max_rgroups], 
 		  gprsos[SuperGlobals.max_rgroups], gprstd[SuperGlobals.max_rgroups], prescribedfire[SuperGlobals.max_rgroups],
 		  spp[SuperGlobals.max_spp_per_grp * SuperGlobals.max_rgroups],
-		  indv[SuperGlobals.max_spp_per_grp * SuperGlobals.max_rgroups];
+		  indv[SuperGlobals.max_spp_per_grp * SuperGlobals.max_rgroups], graze[SuperGlobals.max_rgroups];
```

Comment:
@alexisBelt Shouldn't this be in `gprsos` instead of `indv` as it is a rgroup variable not individual variable?

Reply:
This line is just establishing variables to be used in the function. I can move it next to gprsos to look more organized?